cmake/SSGCommon.cmake

```diff
@@ -1138,3 +1138,19 @@ macro(ssg_build_zipfile ZIPNAME)
         DEPENDS "${CMAKE_BINARY_DIR}/zipfile/${ZIPNAME}.zip"
     )
 endmacro()
+
+macro(ssg_build_nist_zipfile ZIPNAME)
+    add_custom_command(
+        OUTPUT "${CMAKE_BINARY_DIR}/nist-zipfile/${ZIPNAME}-nist.zip"
+        COMMAND ${CMAKE_COMMAND} -E remove_directory "nist-zipfile/"
+	COMMAND ${CMAKE_COMMAND} -E make_directory "nist-zipfile/${ZIPNAME}"
```

Comment:
indents

Reply:
@mpreisler thanks. I think my editor is adding tabs instead of spaces when it auto-indents. Fixed.